lib/checks/shared/has-alt.js

```diff
@@ -1,4 +1,6 @@
-var nn = node.nodeName.toLowerCase();
-return (
-	node.hasAttribute('alt') && (nn === 'img' || nn === 'input' || nn === 'area')
-);
+const { nodeName } = virtualNode.props;
+if (['img', 'input', 'area'].includes(nodeName) === false) {
+	return false;
+}
+
+return typeof virtualNode.attr('alt') === 'string';
```

Comment:
VirtualNode has a `hasAttr` function, any reason why you're not using it?

Reply:
Right you are!